packages/flame/test/components/position_component_test.dart

```diff
@@ -228,5 +228,61 @@ void main() {
       );
       expect(child.absoluteCenter, parent.position + child.center);
     });
+
+    test('scaled component contains point', () {
```

Comment:
We should check our current coverage and increase it if we can 😉

Reply:
Good idea, increased it